templates/mysqlbackup.sh.erb

```diff
@@ -70,18 +70,18 @@ cleanup
 <% if @file_per_database -%>
 mysql --defaults-file=$TMPFILE -s -r -N -e 'SHOW DATABASES' | while read dbname
 do
-  mysqldump --defaults-file=$TMPFILE --opt --flush-logs --single-transaction \
+  mysqldump --defaults-extra-file=$TMPFILE --opt --flush-logs --single-transaction \
```

Comment:
What versions of mysql support the `--defaults-extra-file`?

More so what version was it introduced? And what is the use case? Could you elaborate?